src/client/datascience/export/exportInterpreterFinder.ts

```diff
@@ -24,6 +24,13 @@ export class ExportInterpreterFinder {
 
         // If an interpreter was not passed in, work with the main jupyter interperter
         const selectedJupyterInterpreter = await this.jupyterInterpreterService.getSelectedInterpreter();
+
+        // This might be the first time user needs nbconvert if they are using raw kernel, so allow to select a jupyter interpreter
+        // to install nbconvert into
+        if (!selectedJupyterInterpreter) {
```

Comment:
Reason I wasn't reproing initially is that even if you remove nbconvert the old code works as long as you have "something" set as global Jupyter interpreter, it will just prompt to install it. You have to be in the state with undefined as selected interpreter (if you have a fresh install + raw kernel then 'nbconvert' export might be the first time you need the jupyter global interpreter set.